torch/csrc/jit/python_ir.cpp

```diff
@@ -120,6 +120,21 @@ void initPythonIRBindings(PyObject * module_) {
     .CREATE_ACCESSOR(Graph,g)
     .CREATE_ACCESSOR(Graphs,gs)
 #undef CREATE_ACCESSOR
+    .def("z_",[](Node & n, const char * name, at::Tensor v) {
+        return n.t_(stringToSymbol(name), std::move(v.view({})));
+    })
+    .def("z",[](Node & n, const char * name) {
+        return n.t(stringToSymbol(name));
```

Comment:
This is awkward, because now you can manufacture Python side tensors that have zero dim and one elem, before the Python frontend properly supports it ;)